tests/end_to_end/conftest.py

```diff
@@ -105,9 +105,16 @@ def generate_events(request, metadata):
         events_playbook_path = os.path.join(getattr(request.module, 'test_data_path'), 'playbooks', playbook)
 
         parameters = {'playbook': events_playbook_path, 'inventory': inventory_playbook}
+
         # Check if the test case has extra variables to pass to the playbook and add them to the parameters in that case
+        # Also, add the module extra vars if it is configured.
+        module_extra_vars = getattr(request.module, 'configuration_extra_vars', None)
         if 'extra_vars' in metadata:
             parameters.update({'extravars': metadata['extra_vars']})
+            if module_extra_vars is not None:
+                parameters['extravars'].update(module_extra_vars)
+        elif module_extra_vars is not None:
+            parameters.update({'extravars': module_extra_vars})
```

Comment:
Extra vars for generating events should be in the test case metadata.

Reply:
Done in [df2a57b](https://github.com/wazuh/wazuh-qa/pull/3115/commits/df2a57b36e82d86c756ab4da90e286cc955d3799)